### PR TITLE
fix(ui): fix settingsLayout so that truncate will work in child panels

### PR DESCRIPTION
**Pull request by ckj**
This was happening when long strings were present in any child of `<Content>`

screen shot 2018-10-03 at 5 53 26 pm

This change fixes that and also converts this particular view to use Styled components instead of less.

screen shot 2018-10-03 at 5 53 56 pm

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx [hunk 1]
```diff
@@ -23,22 +25,23 @@ class GroupTombstoneRow extends React.Component {
       actor = data.actor;
 
     return (
-      <li className={`group row level-${data.level} type-${data.type}`}>
-        <div className="col-md-10 event-details issue">
+      <PanelItem align="center">
+        <Box flex="1" style={{minWidth: 0}}>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx]: Should this be a `Content` element instead?

File: src/sentry/static/sentry/less/sentry.less [hunk 2]
```diff
@@ -35,7 +35,6 @@
 @import url('./admin.less');
 @import url('./auth.less');
 @import url('./dashboard.less');
-@import url('./discarded-groups.less');
```
Comment (markstory) [inline src/sentry/static/sentry/less/sentry.less]: :clap:

File: src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx [hunk 3]
```diff
@@ -97,28 +100,24 @@ class GroupTombstones extends AsyncComponent {
...
-                    />
-                  );
-                })}
-              </ul>
-            ) : (
-              this.renderEmpty()
-            )}
-          </div>
-        </div>
+      <div className="discarded-groups">
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx]: Is this class name still in use?

File: src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx (hunk 1 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx]: @markstory if you're talking about `Content` in `settingsLayout.jsx`, then I don't think so. It's a one-off styled component that does something similar, but I don't think it's worth making it its own component and importing it in.

Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx]: Can we create a local styled component at the very least? I'd like to see less inline styles (unless it's a hassle overriding existing CSS with emotion). We

File: src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx (hunk 3 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx]: Nope! Removed.

**Review by billyvg — APPROVED** [review]

**Pull request merged**